### PR TITLE
Open collections if they change

### LINKED ISSUE #51: If pasting a book modified collections, they should be expanded
Achieve this by programatically clicking "show collections" button so we get the nice animation too

### PR DESCRIPTION
**Pull request by braxtonhall**
resolves #51 

It's not pretty but it works lol

### CODE REVIEW

File: extension/ts/objects/bookForm.ts [hunk 1]
```diff
@@ -59,10 +59,22 @@ const insertFormData = (saveData: FormData) => getFormElements()
...
+	// LibraryThing collections checkboxes are sometimes hidden beneath a div that is not visible
+	// THIS IS BRITTLE and relies on the specific markup tree of LibraryThing
+	if (element.id.startsWith(COLLECTIONS_ID_PREFIX)) {
+		const greatGrandParent = element?.parentElement?.parentElement?.parentElement;
+		if (greatGrandParent?.style?.display === "none") {
+			greatGrandParent.style.display = "";
+		}
+	}
+};
+
```
Comment (adirar111) [inline extension/ts/objects/bookForm.ts:77]: oh this is DISGUSTING but hey whatchagonnado

**Review by adirar111 — APPROVED** [review]
open sesame

File: extension/ts/objects/bookForm.ts (hunk 1 above)
Comment (braxtonhall) [reply extension/ts/objects/bookForm.ts:77]: actually idk is there a better way? i don't really know frontend lol

I could a selector from the top but it's! technically just as brittle :(

Comment (adirar111) [reply extension/ts/objects/bookForm.ts:77]: idk either lmao, think it's just necessarily brittle 🤷

Comment (braxtonhall) [reply extension/ts/objects/bookForm.ts:77]: so the alternative is `element.closest(query)`, but it's just as brittle.

The code might look a little cleaner though

**Pull request merged**